build.sbt

```diff
@@ -1,6 +1,21 @@
+inThisBuild(List(
+  organization := "io.github.conduktor",
```

Comment:
why had the organization changed?
GitHub's domain is `github.com`, not? 
Why did you remove the `.ksm`?

Reply:
according to https://central.sonatype.org/publish/requirements/coordinates/ the schema is io.github.username